packages/camera/camera_android_camerax/example/android/app/src/androidTest/java/io/flutter/plugins/cameraxexample/InstanceManagerTest.java

```diff
@@ -8,12 +8,9 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNull;
 
-import androidx.test.ext.junit.runners.AndroidJUnit4;
 import io.flutter.plugins.camerax.InstanceManager;
 import org.junit.Test;
-import org.junit.runner.RunWith;
 
-@RunWith(AndroidJUnit4.class)
```

Comment:
Does this test run for you when you remove this line? When I ran this test with the tool command:

```
dart run script/tool/bin/flutter_plugin_tools.dart native-test --android --no-unit --packages=camera_android_camerax
```

it didn't run this test without it?

Reply:
I will try it, but I removed it because this dependency couldn't be found when I tried running it locally. Have you run into that?

Reply:
I reverted this change. I was able to get it to run!